package/cpp/api/JsiSkColor.h

```diff
@@ -26,6 +26,35 @@ namespace RNSkia {
 
         ~JsiSkColor() {}
 
+        static jsi::Object toValue(jsi::Runtime &runtime, SkColor color) {
+            auto result = runtime
+                    .global()
+                    .getPropertyAsFunction(runtime, "Float32Array")
+                    .callAsConstructor(runtime, 4)
+                    .getObject(runtime);
+            jsi::ArrayBuffer buffer = result
+                    .getProperty(runtime, jsi::PropNameID::forAscii(runtime, "buffer"))
+                    .asObject(runtime)
+                    .getArrayBuffer(runtime);
+            auto bfrPtr = reinterpret_cast<float *>(buffer.data(runtime));
+            auto color4f = SkColor4f::FromColor(color).array();
+            std::copy(color4f.begin(), color4f.end(), bfrPtr);
+            return result;
+        }
+
+        static SkColor fromValue(jsi::Runtime &runtime, const jsi::Value &obj) {
+            const auto& object = obj.asObject(runtime);
+            jsi::ArrayBuffer buffer = object
+                    .getProperty(runtime, jsi::PropNameID::forAscii(runtime, "buffer"))
+                    .asObject(runtime)
+                    .getArrayBuffer(runtime);
+            auto bfrPtr = reinterpret_cast<float *>(buffer.data(runtime));
```

Comment:
Should we have a fallback to regular Arrays here? So that if we pass a JS Array like [0xff, 0x00, 0x00, 0x01] that is not an ArrayBuffer it will work as well? The optimisation could be implemented so that we first try the ArrayBuffer stuff and then the regular array as fallback. On the other hand this might also be taken care of by being extra tight with types in TS.